src/script/entity/message/E2EIVerificationMessage.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {replaceLink, t} from 'Util/LocalizerUtil';
+
+import {SystemMessage} from './SystemMessage';
+
+import {Config} from '../../Config';
+import {SystemMessageType} from '../../message/SystemMessageType';
+
+export class E2EIVerificationMessage extends SystemMessage {
+  constructor() {
+    super();
+    this.system_message_type = SystemMessageType.E2EI_VERIFIED;
+    this.caption = t(
+      'tooltipConversationAllE2EIVerified',
+      {},
+      replaceLink(Config.getConfig().URL.SUPPORT.E2EI_VERIFICATION),
+    );
+  }
```

Comment:
This was the old way of diplaying caption (back when we used `knockout`). Let's render the string directly in the react component now :)